client/src/components/GroupCard/GroupCard.test.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import { shallow, configure, mount } from 'enzyme'
 import GroupCard from './GroupCard.js'
 import Adapter from 'enzyme-adapter-react-16';
+import {BrowserRouter} from 'react-router-dom';
 configure({adapter: new Adapter()});
 import {BrowserRouter} from 'react-router-dom';
```

Comment:
Do we need to import BrowserRouter twice?

Reply:
Yea, this makes the tests fail. Did you run the tests before requesting a review?